.github/workflows/build-scan-push.yml

```diff
@@ -0,0 +1,98 @@
+name: BUILD-SCAN-PUSH
+
+on:
+  push:
+    branches: [ '*' ]
+
+jobs:
+
+  get-matrix-values:
+    runs-on: ubuntu-latest
+    outputs:
+      image: ${{ steps.set-var.outputs.image }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: set-var
+        run: |
+          content=`cat ./image-matrix.json`
+          content="${content//'%'/'%25'}"
+          content="${content//$'\n'/'%0A'}"
+          content="${content//$'\r'/'%0D'}"
+          echo "::set-output name=image::$content"
```

Comment:
Ugh!

This is unavoidable and you have my sympathies for having to implement it

But still, ugh!

😃

Reply:
It made me sad, perhaps there will be better support for reading settings from JSON files in future!